frontend/packages/noobaa-storage-plugin/src/components/health-card/health-card.tsx

```diff
@@ -53,13 +52,16 @@ const getObjectStorageHealthState = (
   const unhealthyBuckets = getGaugeValue(unhealthyBucketsResponse);
   const pools = getGaugeValue(poolsResponse);
   const unhealthyPools = getGaugeValue(unhealthyPoolResponse);
+
   const result: ObjectStorageHealth = {
     message: 'Object Storage is healthy',
     state: HealthState.OK,
   };
 
-  let value;
-  if (_.isEmpty(noobaaSystemData)) {
+  const noobaaSystemData = _.get(noobaaSystem, 'data[0]', null) as K8sResourceKind;
+  const noobaaPhase = _.get(noobaaSystemData, 'status.phase');
+
+  if (!noobaaSystemData || (noobaaPhase && noobaaPhase !== 'Ready')) {
```

Comment:
if noobaPhase is undefined then this will break.
```suggestion
  if (!noobaaSystemData || (noobaaPhase !== 'Ready')) {
```

Reply:
Will not break actually, put on purpose, but not required now.
I have made status `Unavailable` when there is error.
Its good to use now `noobaaPhase !== 'Ready'`.